pkg/base1/cockpit.js

```diff
@@ -1379,7 +1387,9 @@ function basic_scope(cockpit, jquery) {
 
         var ret;
 
-        if (suffix)
+        if (string_representation === "")
+            ret = [];
```

Comment:
Makes sense. Now returns ```[ "" ]``` here. It makes no sense to return a suffix here, and that is a minor change. Defining this behavior can be considered part of stabilizing this API.